tests/unit_tests/data_steward/validation/participants/writers_test.py

```diff
@@ -45,13 +50,20 @@ def test_write_to_result_table(self, mock_load_csv, mock_upload, mock_wait,
                                      matches)
 
         # post conditions
-        self.assertEqual(mock_upload.call_count, 1)
         self.assertEqual(mock_load_csv.call_count, 1)
         self.assertEqual(mock_wait.call_count, 1)
+        self.assertEqual(mock_storage_client.get_bucket.call_count, 1)
+        self.assertEqual(mock_client_bucket.blob.call_count, 1)
+        self.assertEqual(mock_client_blob.upload_from_file.call_count, 1)
 
         upload_path = self.dataset + '/intermediate_results/' + self.site + '.csv'
         self.assertEqual(
-            mock_upload.assert_called_with(bucket_name, upload_path, ANY), None)
+            mock_storage_client.get_bucket.assert_called_with(bucket_name),
+            None)
+        self.assertEqual(
+            mock_client_bucket.blob.assert_called_with(upload_path), None)
+        self.assertEqual(
+            mock_client_blob.upload_from_file.assert_called_with(ANY), None)
```

Comment:
You don't need to wrap these in assertEquals. Same for other instances of this.
```suggestion
        mock_client_bucket.blob.assert_called_with(upload_path)
        mock_client_blob.upload_from_file.assert_called_with(ANY)
```